pkg/controller/kieapp/defaults/merge.go

```diff
@@ -171,13 +171,11 @@ func mergeSpec(baseline appsv1.DeploymentConfigSpec, overwrite appsv1.Deployment
 		return appsv1.DeploymentConfigSpec{}, err
 	}
 	overwrite.Template = mergedTemplate
-
 	mergedTriggers, err := mergeTriggers(baseline.Triggers, overwrite.Triggers)
 	if err != nil {
 		return appsv1.DeploymentConfigSpec{}, err
 	}
 	overwrite.Triggers = mergedTriggers
-
```

Comment:
Ditto

Reply:
Also, we've had a lot of updates, so please rebase. You may be better off manually applying your changes on top of the latest (on this same PR/commit)

Reply:
Done